pylzy/tests/scenarios/repeated_execs_use_cache/expected_stdout

```diff
@@ -0,0 +1,5 @@
+foo was called
+bar was called
+message from bar: number is 42
+bar was called
```

Comment:
is it normal that the `bar` function was called again?

Reply:
yes, bar function has cache=False